src/EventManager.ts

```diff
@@ -72,7 +79,7 @@ export default class EventManager {
      * @param  {string} eventName
      * @param  {function} handler
      */
-    addEventHandler(eventName, handler) {
+    addEventHandler(eventName: string, handler: (event: any) => void): void {
```

Comment:
`event` type should be `ChangeEvent<ChangeObject>` where 

`export type ChangeObject = PermissionChange | SubscriptionChange | EmailSubscriptionChange | SMSSubscriptionChange;`

Reply:
I tried to use more specific event types for these `ChangeEvent` as well as the other class of events 

> `export type HandlerEvent = NotificationReceivedEvent | OpenedEvent | InAppMessageAction` 

I mentioned a little bit in the Paper that I didn't get it working.
This is error I see in `index.ts`

![Screen Shot 2021-11-29 at 12 56 08 PM](https://user-images.githubusercontent.com/4022291/143942804-ed0b3c2c-f659-40c3-a77b-e244285f3c9d.png)

Reply:
Done, `event` is `ChangeEvent<T>`